pages/linux/nl.md

```diff
@@ -0,0 +1,21 @@
+# nl
+
+> A utility for numbering lines, either from a file, or from standard 
+input.
+
+- Number lines in a file:
+
+`nl ./file`
```

Comment:
please use the token format per the guidelines [here](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#token-syntax), particularly point # 3.